Units/parser-c.r/bug1458930.c.d/expected.tags

```diff
@@ -1,2 +1,2 @@
-x	input.c	/^char x();$/;"	p	file:
-y	input.c	/^wchar_t y();$/;"	p	file:
+x	input.c	/^char x();$/;"	p
```

Comment:
file: field is dropped.
The old parser puts file: to the all prototypes.(qualifyFunctionDeclTag)
How do you think about this?

One of view is that a declaration in a *.c file has no impact on the other files as far as the .c file is not included.
Therefore the declaration should have file:.
Another view is that a declaration is not a definition. Ideally it should be tagged as a reference tag, and have functoin kind with declaration role. Therefore thinking about its scope is meaning less.